### PR TITLE
chore: Make bundle compatible with Pimcore 10.2.x

### PR DESCRIPTION
**Pull request by jorisros**

### CODE REVIEW

File: src/Controller/DefaultController.php [hunk 1]
```diff
@@ -28,11 +28,11 @@ public function indexAction(\DataDictionaryBundle\Graph\Graph $graph)
         $graphViz = new GraphViz($graph);
 
         return $this->render(
-            "DataDictionaryBundle:Default:index.html.php",
+            "@DataDictionary/index.html.twig",
```
Comment (kjkooistra-youwe) [inline src/Controller/DefaultController.php:31]: 'Default' section is no longer needed?

File: src/Resources/views/index.html.twig [hunk 2]
```diff
@@ -6,10 +6,8 @@
 </head>
 
 <body>
-<a href="<?php echo $this->path("dataDictonaryImage") ?>" target="_blank">Open image in a new tabs</a>
-<?php
-echo $this->image;
-?>
+<a href="{{ path("dataDictonaryImage") }}" target="_blank">Open image in a new tabs</a>
```
Comment (kjkooistra-youwe) [inline src/Resources/views/index.html.twig:9]: Optionally make this an admin translation.

**Review by kjkooistra-youwe — APPROVED** [review]

**Review by youwe-petervanderwal — APPROVED** [review]
No further remarks than @kjkooistra-youwe already mentioned

File: src/Controller/DefaultController.php (hunk 1 above)
Comment (jorisros) [reply src/Controller/DefaultController.php:31]: No since 3.4 that has been changed. I could add a extra directory but wasnt needed

**Pull request merged**